pytest/tests/mocknet/helpers/neard_runner.py

```diff
@@ -489,11 +489,22 @@ def do_update_config(self, key_value):
 
         return True
 
-    def do_start(self):
+    def do_start(self, send_interval_millis=None):
+        if send_interval_millis is not None and not isinstance(
+                send_interval_millis, int):
+            raise ValueError(
+                f'send_interval_millis: {send_interval_millis} not an int')
         with self.lock:
             state = self.get_state()
             if state == TestState.STOPPED:
-                self.start_neard()
+                if send_interval_millis is not None and not self.is_traffic_generator(
+                ):
+                    logging.warn(
+                        f'got send_interval_millis = {send_interval_millis} on non traffic generator node. Ignoring it.'
+                    )
+                    send_interval_millis = None
+                # TODO: restart it if we get a different send_interval_millis than last time
```

Comment:
I do not get this, you are starting neard below anyways right? what needs to be restarted and why?

Reply:
If you run `mirror.py start-traffic` when the traffic generator is already running, it won't do anything, because in `do_start()`, we only call `start_neard()` if it's currently stopped. So with this PR, if you want to change the value of `send_interval_millis`, you'll need to run `stop-traffic` first, and then start it again. So could be an easier user experience to not require an explicit `stop-traffic` to change the value of this flag for the currently running `neard mirror run` process